string sequence_name
string[] input_types
# Types to be updated with proper GMM/GP containers
KeyValPair[] effect_prob
KeyValPair[] effect_pred
Motion[] motions
